public/app/features/dashboard/state/initDashboard.ts

```diff
@@ -198,6 +199,15 @@ export function initDashboard(args: InitDashboardArgs): ThunkResult<void> {
       console.log(err);
     }
 
+    const left = storeState.explore && storeState.explore.left;
+    if (left && left.originPanelId) {
+      const panelArrId = dashboard.panels.findIndex(panel => panel.id === left.originPanelId);
+      if (panelArrId > -1) {
+        dashboard.panels[panelArrId].targets = left.queries.map(query => ({ refId: query.refId, expr: query.expr }));
```

Comment:
Not every datasource stores its query in the `expr`. Some even have complex objects. Can we simply copy this new edited object?